templates/tomcat/server.xml.erb

```diff
@@ -78,7 +78,7 @@
                SSLEnabled="true"
                maxThreads="150" scheme="https" secure="true"
                clientAuth="want"
-               sslProtocols="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join(",") %>"
+               sslProtocol="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join(",") %>"
```

Comment:
```suggestion
               sslProtocol="<%= scope['::candlepin::tls_versions'].map { |version| "TLSv#{version}"}.join("+") %>"
```

I believe this is the form that is expected in the server.xml conf file

Reply:
Good catch! I see that in examples as well.